src/Runner.Worker/ExecutionContext.cs

```diff
@@ -276,6 +277,9 @@ public void RegisterPostJobStep(IStep step)
             }
             step.ExecutionContext.ExpressionValues["env"] = envContext;
 
+            // Set GITHUB_ACTION_PATH for each step
+            step.ExecutionContext.SetGitHubContext("action_path", actionDirectory);
```

Comment:
we don't have to remove it now since we just set it at the run step level for each step.

Reply:
(this will stomp over each step [we will need to revert and then pop off the action_path]).